### ISSUE TITLE
How to update memcached cache?

### ISSUE THREAD

**Issue by joeljv91** [browscap/browscap-php#169]
Hi

I'm using this library in my project but I don't know how to update cache. I can't use cli command because I'm using Memcached

This is my testing code:

```
$memcacheConfiguration = [
    'host'             => 'localhost',
    'port'             => 11211,
    'namespace'        => 'wurfl',
    'cacheExpiration'  => 0,
    'cacheVersion'     => '1234',
];

$adapter = new WurflCache\Adapter\Memcached($memcacheConfiguration);
$bcUpdater = new BrowscapPHP\BrowscapUpdater();
$bcUpdater->update();
```

But this doesn't work. How can I generate and save cache to memcached?

Thanks

**Comment by mimmi20** [comment]
Do you get any error message? How do you know It didn't work?

**Comment by asgrim** [comment]
Providing some more information other than "it doesn't work" would really help... Otherwise this is just gonna get closed.

**Comment by joeljv91** [comment]
Well, when I try to access to the "generated" cache with my memcached configuration shows this exception:

```
Exception information:
Message: there is no active cache available, please run the update command
Stack trace:
#0 /var/www/phentermine.com/application/controllers/IndexController.php(54): BrowscapPHP\Browscap->getBrowser()
```

This is my testing code:

```
$memcacheConfiguration = [
    'host'             => 'localhost',
    'port'             => 11211,
    'namespace'        => 'wurfl',
    'cacheExpiration'  => 0,
    'cacheVersion'     => '1234',
];

$adapter = new WurflCache\Adapter\Memcached($memcacheConfiguration);
$bcUpdater = new BrowscapPHP\BrowscapUpdater();
$bc = new BrowscapPHP\Browscap();

$bcUpdater->setCache($adapter);
$bcUpdater->update();

$bc->setCache($adapter);
$bcData = $bc->getBrowser();
```

I'm going to add some logs and try to get more information

**Comment by mimmi20** [comment]
Do you get error messages during the update?

**Comment by joeljv91** [comment]
This is the logger data when I update cache with memcached config:

```
[2016-08-04 05:52:27] Test.DEBUG: started fetching remote file [] []
[2016-08-04 05:52:27] Test.INFO: there is no cached version available, please update from remote [] []
[2016-08-04 05:52:28] Test.DEBUG: finished fetching remote file [] []
[2016-08-04 05:52:28] Test.INFO: start creating patterns from the ini data [] []
[2016-08-04 05:52:50] Test.INFO: finished creating patterns from the ini data [] []
[2016-08-04 05:52:50] Test.INFO: start creating data from the ini data [] []
[2016-08-04 05:52:59] Test.INFO: finished creating data from the ini data [] []
```

**Comment by joeljv91** [comment]
Debugging the library I notice that the "cacheExpiration" has the value "315360000" not 0 and I set 0 in my memcached configuration

**Comment by asgrim** [comment]
Fixed by #170
